requirements.txt

```diff
@@ -1,5 +1,5 @@
-# tensorflow==2.4.1
-tensorflow_addons
+tensorflow

```

Comment:
Could you comment out `tensorflow`? Actually, it messes up with CUDA in Kaggle in different environments. So let's leave it to the users to manually install it.

Reply:
Oh, sure thing! Will do it for both PRs.

Reply:
Done.